File: app/src/main/java/com/van/shortcuts/ShortcutReceiver.java
package com.van.shortcuts;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

public class ShortcutReceiver extends BroadcastReceiver {

    private static final String TAG = "ShortcutReceiver";

    @Override
    public void onReceive(Context context, Intent intent) {
        Log.i(TAG, "onReceive : " + intent.toString());
    }
}

File: app/src/main/java/com/van/shortcuts/ShortcutReceiver.kt
package com.van.shortcuts

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

class ShortcutReceiver : BroadcastReceiver() {

    companion object {
        private const val TAG = "ShortcutReceiver"
    }

    override fun onReceive(context: Context, intent: Intent) {
        Log.i(TAG, "onReceive : $intent.toString()")
    }
}
